play-json/src/test/scala/com/lightbend/lagom/scaladsl/playjson/PlayJsonSerializerSpec.scala

```diff
@@ -136,6 +136,9 @@ case class Box(surprise: Option[String])
 
 class PlayJsonSerializerSpec extends WordSpec with Matchers {
 
+  // this is a magic number. I know longContent will compress well under this 1024.
+  val COMPRESSION_THRESHOLD = 32 * 1024
```

Comment:
```suggestion
// this is a magic number. I know longContent will compress well under this 32 * 1024.
```

Reply:
I think the suggestion is off by a row. I think I understand that you intend to fix the comment (not remove the `val`. I'll fix locally and push.

Reply:
yes, you are right